### PR TITLE
feat: gas price

### PR DESCRIPTION
**Pull request by greged93**
Adds the gas price endpoint.

Time spent on this PR: 0.5 day

Resolves: #171 

# Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Testing

# What is the new behavior?
Gas price route is implemented and tested.

# Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by greged93** [comment]
The gas price is queried using 2 contracts each deployed on every networks. The contracts consist of one account contract and one simple counter contract. 
# Mainnet
- Counter: https://starkscan.co/contract/0x02786c4cdfb2ee39727cb00695cf136710e2c3bfc5cb09315101be3d37c2c557
- Account: https://starkscan.co/contract/0x07142fbf6e8c9c07b079d47727c6d2ff49970203bfd5bd6ed0d740e0f5a344e7

# Testnet 1
- Counter: https://testnet.starkscan.co/contract/0x03c12643f0e9f0b41de95a87e4f03f5fa69601930e9354a206a0b82a02119f2b
- Account: https://testnet.starkscan.co/contract/0x07142fbf6e8c9c07b079d47727c6d2ff49970203bfd5bd6ed0d740e0f5a344e7

# Testnet 2
- Counter: https://testnet-2.starkscan.co/contract/0x00e438661a4775fdf10cf132cc50730f40e59f3d040b15e64cd292add25eb01b#read-write-contract
- Account: https://testnet-2.starkscan.co/contract/0x07142fbf6e8c9c07b079d47727c6d2ff49970203bfd5bd6ed0d740e0f5a344e7#overview

File: crates/core/src/models/call.rs [hunk 1]
```diff
@@ -2,22 +2,52 @@ use std::slice::SliceIndex;
...
+impl From<StarknetCall> for Call {
+    fn from(call: StarknetCall) -> Self {
+        Self(call)
+    }
+}
+
+impl From<Call> for Vec<FieldElement> {
+    fn from(call: Call) -> Self {
+        let mut c = vec![
+            FieldElement::ONE,
```
Comment (danilowhk) [inline crates/core/src/models/call.rs:29]: Maybe we could add some comments on the structure of how a vector calldata is constructed? Got a bit confused of why the "One" and then the "Zero"

File: crates/core/src/client/constants.rs [hunk 2]
```diff
@@ -77,3 +81,23 @@ lazy_static! {
 lazy_static! {
     pub static ref KAKAROT_CLIENT_VERSION: String = format!("kakarot_{}", env!("CARGO_PKG_VERSION"));
 }
+
+// This module contains constants related to transactions used to calculate
+// Starknet gas price
+lazy_static! {
+    pub static ref ACCOUNT_ADDRESS: FieldElement =
+        FieldElement::from_hex_be("0x07142FbF6E8C9C07b079D47727C6D2ff49970203bfd5Bd6ED0D740e0f5a344E7").unwrap();
```
Comment (danilowhk) [inline crates/core/src/client/constants.rs:89]: Maybe we could add the codes of these "demo" contracts so we are just aware of their code?

**Comment by danilowhk** [comment]
Lgtm!

**Review by danilowhk — APPROVED** [review]

**Pull request merged**